Configurations/descrip.mms.tmpl

```diff
@@ -311,8 +311,8 @@ PERLASM_SCHEME={- $target{perlasm_scheme} -}
 CPPFLAGS_Q={- (my $x = $cppflags) =~ s|"|""|g;
               (my $d = $defines) =~ s|"|""|g;
               $x .= "/INCLUDE=($includes)" if $includes;
-              $x .= "/DEFINE=($d)";
-              $x; -}
+              $x .= "/DEFINE=($d)" if $defines;
+              $x; -} 
```

Comment:
There is redundant/invisible space at the end of this line. If you care that is. Just in case, I don't really look at descrip.mms.tmpl. This simply caught the eye in git log -p

Reply:
Thanks.